lib/opensusebasetest.pm

```diff
@@ -909,6 +909,9 @@ sub wait_boot {
     # When no bounce back on power KVM, we need skip bootloader process and go ahead when 'displaymanager' matched.
     elsif (get_var('OFW') && (check_screen('displaymanager', 5))) {
     }
+    # SLE16 ppc too fast grub2 run / directly ready to login
```

Comment:
Let's improve the wording a bit and specify the correct name for the arch:

```suggestion
    # SLE16 ppc64le is too fast from grub2 run to ready to login
```

Reply:
Commit message updated too

Reply:
Here **PPC** means itself **PowerPC** as a CPU category, like also x64 is when compared with x86_64, focusing the discussion orientation on that type of cpu, not only ppc64le.
PPC is not different but includes ppc64le. This told, I'll use the full name.
https://en.wikipedia.org/wiki/PowerPC